README.md

```diff
@@ -22,7 +25,8 @@ To get a local copy up and running follow these simple example steps.
 - Clone or download this repo on your machine
   
   ```git clone git@github.com:Kalunge/Blog-app.git```
-- cd into project directory
+- cd into project directory![145261309-cf2fa744-ddfb-4c37-bf6b-5a5ebf6ea28d](https://user-images.githubusercontent.com/50773868/145261402-cc401dbf-f42f-4b09-b291-5605678952ee.png)
+
```

Comment:
- [ ] It would be great if you add instructions on how to create a user since there's no interface for that in your application

Reply:
Well solved